tests/CMakeLists.txt

```diff
@@ -154,3 +157,9 @@ endforeach()
 target_link_libraries(dispatch_group PRIVATE m)
 target_link_libraries(dispatch_timer_short PRIVATE m)
 
+# test-specific compiler flags
+target_compile_options(dispatch_c99 PRIVATE -std=c99)
+
+# test-specific compiler flags (for currently disabled tests)
+#add_unit_test(dispatch_priority2 SOURCES dispatch_priority.c)
+#target_compile_options(dispatch_priority2, PRIVATE -DUSE_SET_TARGET_QUEUE=1)
```

Comment:
Drop the `,`.  Why is this test disabled?  We do have the `DISABLED_TEST` keyword, so we can add the test but not have it be built/run.

    add_unit_test(dispatch_priority2
                  DISABLED_TEST
                  SOURCES
                    dispatch_priority.c)

I think that we should add a dummy target so we can actually add the additional flags.